### DISCUSSION TITLE
Zone import of existing PDNS server

### DISCUSSION THREAD

**Discussion by SSchneebeli** [ngoduykhanh/PowerDNS-Admin#1293] [Q&A] [answered]
Hi

I'm running a PowerDNS Server with several zones and would like to use PowerDNS-Admin but not start over from scratch and recreate all zones.
Is there a possibility to import the existing zones in to PowerDNS-Admin?

Thanks,
Stefan

**Comment by arbaldin** [comment]
Hi @SSchneebeli.

PDNS-Admin is not a DNS itself, it's a web interface to manage your PDNS. It's another piece of software that access your PDNS installation through the PDNS API (that you should enable).

Once you install PDNS-Admin and configure it pointing to your PDNS IP (which have the API enabled) it should work and read all zones you have in your currently PDNS installation.

Regards.

**Comment by SSchneebeli** [comment]
Hi arbaldin,

this is exactly what I did, but the existing zones are not visible. 
I'm using the PDNS API already with Ansible so I'm sure it's working.
Are there any logfiles I can check?

Regards,

**Comment by metrax** [comment, marked as answer]
This depends on how you installed PDA.
If you are using systemd for starting PDA, than you can use
`journalctl-u powerdns-admin`
to check the logs. Sometimes the systemd service has another name, you need to adapt it at the command.

**Reply by SSchneebeli** [reply]
Hi Metrax,

PowerDNS-Admin is running in a docker container. 
Are there any logs inside the container?

**Reply by jbe-dw** [reply]
I guess docker logs <container> would show something. You can try creating a zone in the GUI and see if it show and error a well.

**Reply by jbe-dw** [reply]
Note that you have to add the PowerDNS address and ApiKey in settings/PDNS and use the full scheme and port to point your API like https://mypdnsapi.local:4443

**Reply by SSchneebeli** [reply]
The problem was the API access restrictions in PowerDNS itself, the subnet where PDNS-Admin ist runniung was not allowed to access the API. Now I see all zones and can create new ones or edit existing ones.

Thank you anyway for help!
